demo/src/app/components/+modal/demos/scrolling-long-content/scrolling-long-content.html

```diff
@@ -0,0 +1,17 @@
+<button type="button" class="btn btn-primary" (click)="openModal(template)">Open modal</button>
+<br>
+
+<ng-template #template>
+  <div class="modal-header">
+    <h4 class="modal-title pull-left">Modal</h4>
+    <button type="button" class="close pull-right" aria-label="Close" (click)="modalRef.hide()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <ng-container *ngFor="let item of items">
```

Comment:
no need for ng-container if there's just `<p>` inside. use <p *ngFor="let item of items">